src/MSCorEE.Tests/MSCorEE.Tests.csproj

```diff
@@ -17,12 +17,16 @@
     <AssemblyName>PInvoke.MSCorEE.Tests</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
+    <ResolveComReferenceSilent>true</ResolveComReferenceSilent>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
   <ItemGroup>
+    <COMFileReference Include="$(MSBuildProgramFiles32)\Windows Kits\NETFXSDK\4.6.1\Lib\um\x86\metahost.tlb">
```

Comment:
I wonder if including the tlb in the repo wouldn't be simpler for contributors (One less file external to have installed).

The files are quite small but I don't know if we have the right to distribute them.

The best would be for the SDK guys to publish a nuget for each of them. (But we could do it ourselves, including them in our nuget it would also simplify consumption)